libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -294,14 +306,15 @@ const decorateMenu = (config) => logErrorFor(async () => {
     const content = await res.text();
     const parsedContent = await replaceText(content, getFedsPlaceholderConfig(), undefined, 'feds');
     menuTemplate = toFragment`<div class="feds-popup">
-        <div class="feds-menu-content">
-          ${parsedContent}
+        <div class="feds-menu-container">
+          <div class="feds-menu-content">
+              ${parsedContent}
+          </div>
         </div>
       </div>`;
-
     // Content has been fetched dynamically, need to decorate links
     decorateLinks(menuTemplate);
-    await decorateColumns({ content: menuTemplate.firstElementChild });
+    await decorateColumns({ content: menuTemplate.querySelector('.feds-menu-content') });
```

Comment:
Since we create the `.feds-menu-content` element ourselves, we could avoid querying. Something like:
```
const menuContent = toFragment`<div class="feds-menu-content">${parsedContent}</div>`;
menuTemplate = toFragment`<div class="feds-popup">
  <div class="feds-menu-container">
    ${menuContent}
  </div>
</div>`;
await decorateColumns({ content: menuContent });
```
I didn't check the logic, so it might need some tweaking, but the general idea would be to create both elements and use their references where needed.

Reply:
Done 👍